index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the delay
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
+            </p>
+            <p>
+              The <a><code>latency</code></a> attribute value depends on the
+              platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>latency</code></a> attribute value will be
+              updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimestamp devicePosition
+          </dt>
+          <dd>
+            <p>
+              Holds an <a><code>AudioTimestamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a href=
+              "#widl-AudioTimestamp-contextTime"><code>contextTime</code></a>
```

Comment:
It is just taken from the platform API (for Windows` IAudioClock::GetPosition`), no specific algorithms are required. Pls. see https://github.com/crosswalk-project/chromium-crosswalk/blob/96132b6d03991f931b48f138b83c8f06de9a4578/media/audio/win/audio_low_latency_output_win.cc#L544